tiling.js

```diff
@@ -38,6 +38,9 @@ export const FocusModes = { DEFAULT: 0, CENTER: 1 }; // export
 
 export const CycleWindowSizesDirection = { FORWARD: 0, BACKWARDS: 1 };
 
+// position to open window at (e.g. to the right of current window)
+export const OpenWindowPositions = { RIGHT: 0, LEFT: 1, START: 2, END: 3 };
```

Comment:
Since you have this as basically an enum it would be ideal to refer to this everywhere, where you currently just write 0, 1, 2, 3.

But this is just a nitpick and I don't know if that would actually be convenient/possible.

Reply:
This is a good call.

The reason why I couldn't use in `prefs.js` is because it's was defined in `tiling.js` and we can't import `tiling.js` into `prefs.js` (it has too many deps that depend on gnome-shell thread - not pref.js doesn't run on gnome thread).

In any case, could actually do it if define the enum in `settings.js` (this module is lightweight and doesn't have strong deps so can be used in extensions.js and prefs.js.

I think it's worth doing.

Reply:
See latest commit.